store/sqlstore/post_store.go

```diff
@@ -1232,7 +1232,7 @@ func (s *SqlPostStore) GetPostsByIds(postIds []string) ([]*model.Post, *model.Ap
 	_, err := s.GetReplica().Select(&posts, query, params)
 
 	if err != nil {
-		mlog.Error(fmt.Sprint(err))
+		mlog.Error("Query error getting posts:", mlog.Err(err))
```

Comment:
Probably here we can replace the `:` at the end with a `.`.

Reply:
done